assets/scss/bootstrap-migration/_maps.scss

```diff
@@ -33,3 +33,8 @@ $spacers: map-merge(
   )
 );
 
+$sizes: ();
+@for $i from 1 through 20 {
+    $size: $i * 5;
+    $sizes: map-merge($sizes, ($size: $size * 1%));
+}
```

Comment:
This code is not required after one of fixes, we let's left it because may be helpful in future